packages/web-app-files/src/components/Collaborators/CollaboratorsEditOptions.vue

```diff
@@ -1,13 +1,18 @@
 <template>
   <div>
-    <translate tag="label" for="files-collaborators-role-button" class="oc-label">Role</translate>
+    <translate
+      :aria-label="$gettext('Role')"
+      tag="label"
+      for="files-collaborators-role-button"
+      class="oc-label"
+      >Role</translate
+    >
     <oc-select
       v-model="selectedRole"
       input-id="files-collaborators-role-button"
       class="files-collaborators-role-button-wrapper"
       :options="roles"
       :clearable="false"
-      label="label"
```

Comment:
On https://github.com/owncloud/owncloud-design-system/pull/1530 we have changed the way labels work for the `OcSelect` component so you should be able to remove the span above and pass a computed property in the label line you deleted (see https://github.com/owncloud/owncloud-design-system/pull/1530/commits/088159bbbc8dd7b159210ebc2eea6706c1c58e4a)